radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/tests/consensus_ledger_sync/ByzantineSyncTest.java

```diff
@@ -25,45 +25,68 @@
 import com.radixdlt.counters.SystemCounters.CounterType;
 import com.radixdlt.integration.distributed.IncorrectAlwaysAcceptingAccumulatorVerifierModule;
 import com.radixdlt.integration.distributed.SometimesByzantineCommittedReader;
+import com.radixdlt.integration.distributed.simulation.NetworkDroppers;
+import com.radixdlt.integration.distributed.simulation.NetworkLatencies;
+import com.radixdlt.integration.distributed.simulation.NetworkOrdering;
 import com.radixdlt.integration.distributed.simulation.SimulationTest;
 import com.radixdlt.integration.distributed.simulation.SimulationTest.Builder;
 import com.radixdlt.integration.distributed.simulation.SimulationTest.TestResults;
 import com.radixdlt.ledger.StateComputerLedger.LedgerUpdateSender;
 import com.radixdlt.sync.CommittedReader;
+import java.util.Arrays;
+import java.util.Collection;
 import java.util.LongSummaryStatistics;
 import java.util.concurrent.TimeUnit;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.assertj.core.api.AssertionsForClassTypes;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
 
 /**
  * Any number/sort of byzantine sync modules should never be able to cause
  * a safety failure.
  */
+@RunWith(Parameterized.class)
 public class ByzantineSyncTest {
-	Logger logger = LogManager.getLogger();
-	private final Builder bftTestBuilder = SimulationTest.builder()
-		.numNodes(4)
-		.randomLatency(10, 200)
-		.addByzantineModuleToAll(new AbstractModule() {
-			@Override
-			protected void configure() {
-				Multibinder<LedgerUpdateSender> committedSenders = Multibinder.newSetBinder(binder(), LedgerUpdateSender.class);
-				committedSenders.addBinding().to(SometimesByzantineCommittedReader.class).in(Scopes.SINGLETON);
-				bind(CommittedReader.class).to(SometimesByzantineCommittedReader.class).in(Scopes.SINGLETON);
-				bind(SometimesByzantineCommittedReader.class).in(Scopes.SINGLETON);
-			}
-		})
-		.pacemakerTimeout(5000)
-		.addOneNodeNeverReceiveProposalDropper()
-		.ledgerAndSync()
-		.checkConsensusSafety("safety")
-		.checkConsensusLiveness("liveness", 5000, TimeUnit.MILLISECONDS)
-		.checkConsensusNoTimeouts("noTimeouts")
-		.checkConsensusAllProposalsHaveDirectParents("directParents")
-		.checkLedgerInOrder("ledgerInOrder")
-		.checkLedgerProcessesConsensusCommitted("consensusToLedger");
+	@Parameters
+	public static Collection<Object[]> numNodes() {
```

Comment:
What is the thinking behind this?  Doesn't seem like a good tradeoff between complexity and utility.